install/streams.csv

```diff
@@ -2122,6 +2122,9 @@ SNGZ,10,E,H,100,false,false,false,2009-06-08T03:00:00Z,9999-01-01T00:00:00Z
 SNVZ,10,E,H,100,false,false,false,2015-03-27T02:00:00Z,9999-01-01T00:00:00Z
 SNVZ,30,L,D,1,false,false,false,2020-02-20T04:50:00Z,9999-01-01T00:00:00Z
 SNVZ,30,H,D,100,false,false,false,2020-02-20T04:50:00Z,9999-01-01T00:00:00Z
+SNZO,00,B,H,40,true,false,false,2024-08-05T08:54:00Z,9999-01-01T00:00:00Z
```

Comment:
I know BH streams are available but I thought the intention was to ignore them and only collect HH. One to double check, I'll re-read the ticket in case the scope has changed.

Reply:
BH streams are strored presently
we may keep it like this  as we re using - at the moment - BH for the Non-NZ network stations ( practicla resons)

Reply:
I mean the purpose of this process was to adopt SNZO as a national network site and treat it as such. That would mean it would be the only national network site supplying BH streams but it seems we are ok with that? Were we not only collecting the BH streams historically because HH was not available (and now it has been turned on for us)? I cant find any reference to the sampling rate requirements in existing tickets/programme decision making template but found this comment from Sam:

https://github.com/GeoNet/tickets/issues/16239#issuecomment-2297946122
"data are available at the equivalent sampling rate"

any thoughts? @ozym @staylorofford

Reply:
The current processing system uses BH?.  Until that switches over we will continue to recover BH? and it's easier simply to store what comes in. We could clearly backfill this if needed or wanted.  Once we're switched over to HH? completely then the question could perhaps be asked again (i.e. there will be no picks being made on the SNZO BH? streams).